### ISSUE TITLE
Is it possible to deactivate Jitsi Support at all ?

### ISSUE THREAD

**Issue by Lighti123** [vector-im/riot-web#13379]
Hello!

is it possible to deactivate jitsi support at all?

We have a closed down matrix server. 
We use your Riot Client (which is nice!) but don't want to use jitsi at all.

Until now, as we also have no integration server, there was an error message when trying a conference call which was fine.

Since the better jitsi "integration", conference calls are working with the drawback that data is transmitted outside of our network.

With the web client i could deactivate jitsi with:
```
     "jitsi": {
       "preferredDomain": "",
       "externalApiUrl": ""
     },
```

But with the Desktop (and Smartphone Clients after the current bug is fixed) i can not change the config so i have to rely on  .well-known/matrix/client  for jitsi configuration.

But at the coming  .well-known/matrix/client  configuration for jitsi, i see only "preferredDomain" documented?
Would the following be enough or would it fall back to some default jitsi server?
```
  "im.vector.riot.jitsi": {
    "preferredDomain": ""
  }
```

Or could you for example add an option to  .well-known/matrix/client  like:
```
  "im.vector.riot.jitsi": {
    "useJitsi": "false"
  }
```

Thanks and Best

**Comment by turt2live** [comment]
It's best to ask these questions in #riot-web:matrix.org

It can't currently be disabled, and `externalApiUrl` isn't used anymore, and there's nothing to stop you from using .well-known (when Riot 1.6 is released) to set your `preferredDomain` to `127.0.0.1` or something not-empty so it just fails rather than does non-deterministic behaviour.

Otherwise, this is a duplicate of https://github.com/vector-im/riot-web/issues/5903

**Comment by bpcurse** [comment]
> Otherwise, this is a duplicate of #5903
@turt2live IMHO not, 5903 deals with disabling the video functionality keeping audio conferencing possible, unrelated to the used server.
This issue is a specific privacy concern about adding a third party jitsi server without notice and the missing possibility to deactivate it.
